theano/gpuarray/opt.py

```diff
@@ -292,22 +579,24 @@ def local_gpuaalloc2(node):
                 i.owner.op in [host_from_gpu, tensor.alloc]
                 for i in c.inputs[1:])
             for c, idx in node.outputs[0].clients)):
-        return [host_from_gpu(GpuAlloc(None)(*node.inputs))]
+        return [host_from_gpu(gpu_alloc(None)(*node.inputs))]
 
 
 @register_opt('fast_compile')
 @op_lifter([tensor.Alloc])
-def local_gpuaalloc(node, context_name):
-    return GpuAlloc(context_name)(*node.inputs)
+@register_opt2([tensor.Alloc], 'fast_compile')
+def local_gpuaalloc(op, context_name, inputs, outputs):
+    return gpu_alloc(context_name)(*inputs)
 
 
 @register_opt('fast_compile')
 @op_lifter([tensor.AllocEmpty])
-def local_gpuaallocempty(node, context_name):
+@register_opt2([tensor.AllocEmpty], 'fast_compile')
+def local_gpuaallocempty(op, context_name, inputs, outputs):
     # We use _props_dict() to make sure that the GPU op know all the
     # CPU op props.
-    return GpuAllocEmpty(context_name=context_name,
-                         **node.op._props_dict())(*node.inputs)
+    dtype = op._props_dict().get('dtype')
```

Comment:
Yeah, i remember @nouiz saying about this. I've made the change there, i'll make it here as well